works.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>About</title>
+  <link href="https://fonts.googleapis.com/css?family=Abril+Fatface|Italiana" rel="stylesheet" type='tex/css'>
+  <link rel="stylesheet" href="css/style.css" type="text/css">
+</head>
+
+<body>
+  <hr>
+  <h1 class= "bio">Works</h1>
+    <nav class="bio">
+    <a href="main.html">Home</a>
+    |
+    <a href="bio.html">About</a>
+    |
+    <a href="works.html">Works</a>
+    </nav>
+  <hr>
+  <nav>
+    <a href="https://www.facebook.com/Grownbeans-140778912663452/"><img border="0" alt="facebook" src="file:///Users/claudia/Desktop/FEWD/github/homework-01-famous-person-resume-website/imgs/1486830908_online_social_media_facebook.png" width="100" height="100" id="social"></a>
```

Comment:
The common convention for external links is to open them up in a new tab/window.  🔨